app/swagger/schemas/gibct/institutions.rb

```diff
@@ -0,0 +1,189 @@
+            property :repayment_rate_all_students, type: [:null, :number]
+            property :avg_stu_loan_debt, type: [:null, :number]
+            property :calendar, type: [:null, :string]
+            property :online_all, type: [:null, :string]
+            property :p911_tuition_fees, type: :number
+            property :p911_recipients, type: :integer
+            property :p911_yellow_ribbon, type: :number
+            property :p911_yr_recipients, type: :integer
+            property :accredited, type: :boolean
+            property :accreditation_type, type: [:null, :string]
+            property :accreditation_status, type: [:null, :string]
+            property :complaints, type: :object do
+              property :facility_code, type: :integer
+              property :financial_by_fac_code, type: :integer
+              property :quality_by_fac_code, type: :integer
+              property :refund_by_fac_code, type: :integer
+              property :marketing_by_fac_code, type: :integer
+              property :accreditation_by_fac_code, type: :integer
+              property :degree_requirements_by_fac_code, type: :integer
+              property :student_loans_by_fac_code, type: :integer
+              property :grades_by_fac_code, type: :integer
+              property :credit_transfer_by_fac_code, type: :integer
+              property :credit_job_by_fac_code, type: :integer
+              property :job_by_fac_code, type: :integer
+              property :transcript_by_fac_code, type: :integer
+              property :other_by_fac_code, type: :integer
+              property :main_campus_roll_up, type: :integer
+              property :financial_by_ope_id_do_not_sum, type: :integer
+              property :quality_by_ope_id_do_not_sum, type: :integer
+              property :refund_by_ope_id_do_not_sum, type: :integer
+              property :marketing_by_ope_id_do_not_sum, type: :integer
+              property :accreditation_by_ope_id_do_not_sum, type: :integer
+              property :degree_requirements_by_ope_id_do_not_sum, type: :integer
+              property :student_loans_by_ope_id_do_not_sum, type: :integer
+              property :grades_by_ope_id_do_not_sum, type: :integer
+              property :credit_transfer_by_ope_id_do_not_sum, type: :integer
+              property :jobs_by_ope_id_do_not_sum, type: :integer
+              property :transcript_by_ope_id_do_not_sum, type: :integer
+              property :other_by_ope_id_do_not_sum, type: :integer
+            end
+          end
+
+          property :meta, '$ref': :GibctInstitutionsShowMeta
+        end
+
+        swagger_schema :GibctInstitutionsAutocompleteMeta do
+          key :type, :object
+          key :required, [:version, :term]
+
+          property :version, type: :integer
+          property :term, type: :string
+        end
+
+        swagger_schema :GibctInstitutionsSearchMeta do
```

Comment:
Search meta element contains the facets which are an interesting detail of this API and I think worth documenting here.